src/PKSim.Core/Model/SupportedProtein.cs

```diff
@@ -1,17 +1,18 @@
 ﻿using OSPSuite.Core.Domain;
 
-namespace PKSim.Core.Model;
-
-public class SupportedProtein: IWithName
+namespace PKSim.Core.Model
 {
-   public string Name { get; set; }
-   public string ParameterName { get; }
-   public string TableParameterName { get; }
-
-   public SupportedProtein(string name, string parameterName, string tableParameterName)
+   public class SupportedProtein : IWithName
    {
-      Name = name;
-      ParameterName = parameterName;
-      TableParameterName = tableParameterName;
+      public string Name { get; set; }
+      public string ParameterName { get; }
+      public string TableParameterName { get; }
+
+      public SupportedProtein(string name, string parameterName, string tableParameterName)
+      {
+         Name = name;
+         ParameterName = parameterName;
+         TableParameterName = tableParameterName;
+      }
    }
 }
```

Comment:
this has nothing to do with the PR; however, otherwise it does not compile on my comp in VS2019.
I wonder why it builds on AppVeyor (VS2019 as well!)

Reply:
oh fucking resharper lol. thanks